tests/workflows/fastq_download_prefetch_fasterqdump_sratools/main.nf.test

```diff
@@ -32,6 +32,42 @@ nextflow_workflow {
                 { assert workflow.success },
                 { assert snapshot(workflow.out).match() }
             )
+
+            assert path("$launchDir/output").exists()
+
+            def lines = path("$launchDir/output/prefetch/failures_report.csv").readLines()
+            assert lines.contains("test_single_end,DRR000774").equals(false)
+            assert lines.contains("test_paired_end,SRR11140744").equals(false)
+        }
+    }
+
+    test("Download errors: 403") {
+
+        when {
+            workflow {
+                """
+                input[0] = Channel.of(
+                    [[ id:'SAMPLE1', single_end:false ], 'ERR1109373'],
+                    [[ id:'ERROR1', single_end:false ], 'SRR999908'],
+                    [[ id:'ERROR2', single_end:false ], 'SRR999934'],
+                    [[ id:'SAMPLE2', single_end:false ], 'SRR13191702']
+                )
+                input[1] = []
+                """
+            }
+            params {
+                outdir = "output"
+            }
+        }
+
+        then {
+            assert workflow.success
+            assert path("$launchDir/output").exists()
+
+            def lines = path("$launchDir/output/prefetch/failures_report.csv").readLines()
```

Comment:
Could you add an assert to make sure that these are the only 2 lines in the file?

Reply:
Update in https://github.com/phac-nml/fetchdatairidanext/commit/bbf2e3fe013b902b060af3ca063f5fad25f837fd